site/_posts/2020-06-03-user-registration-and-sign-in-with-laravel.md

```diff
@@ -178,23 +187,44 @@ class RegisterUser extends Controller
     return redirect('/');
   }
 }
+//..
 ```
 
-_Note: FusionAuth includes several other options for user registration. Be sure to [check out the documentation](https://fusionauth.io/docs/v1/tech/apis/registrations) to make sure you include all the appropriate options._
+Finally, you'll need to create two routes to register new users: one to display the registration form and another to process the data and interact with the FusionAuth client. Add the following lines to your `routes/web.php` file (remove the welcome route):
+
+```php
+Route::view('/register', 'register');
+Route::post('/register', 'RegisterUser');
+```
+
+{% include _callout-note.liquid content=
+"FusionAuth includes several other options for user registration. Be sure to [check out the documentation](https://fusionauth.io/docs/v1/tech/apis/registrations) to make sure you include all the appropriate options."
```

Comment:
Maybe `FusionAuth supports different features when a user registers, including verification emails, required fields and custom roles. Be sure to ...`